services/app-web/src/components/HealthPlanPackageTable/HealthPlanPackageTable.module.scss

```diff
@@ -51,3 +51,15 @@
     color: $theme-color-base-ink;
     padding-top: units(3);
 }
+
+.visuallyHidden {
```

Comment:
There is a shared `.srOnly` class that is global. I think this is what we need? I didn't notice the borders sorry about that